src/main.cpp

```diff
@@ -1497,33 +1530,30 @@ main(int argc,
     if ( otumapout_gv ) align_id = 0.97;
     else align_id = 0;
   }
-  
   // default %query coverage to keep alignment
   if ( align_cov < 0 )
   {
     // if OTU-map is chosen, set default coverage to 0.97
     if ( otumapout_gv ) align_cov = 0.97;
     else align_cov = 0;
-  }
-    
+  } 
   // 3. For each window, traverse in parallel the Burst trie/reverse and
   // LEV(k), outputting all reads with edit distance <= k between the
   // window.
-  paralleltraversal( readsfile,
-                     ptr_filetype_ar,
-                     ptr_filetype_or,
-                     match,
-                     mismatch,
-                     gap_open,
-                     gap_extension,
-                     score_N,
-                     skiplengths,
-                     argc,
-                     argv,
-                     yes_SQ,
-                     myfiles,
-                     exit_early);
-  
+  paralleltraversal(readsfile,
+                    have_reads_gz,
```

Comment:
this arguments list is getting large, would it make sense to pass around a struct?